form/src/main/java/org/dhis2/form/ui/provider/inputfield/FieldProvider.kt

```diff
@@ -538,6 +548,48 @@ private fun ProvideInteger(
     )
 }
 
+@Composable
+private fun ProvideInputLink(
+    modifier: Modifier,
+    fieldUiModel: FieldUiModel,
+    intentHandler: (FormIntent) -> Unit,
+    uiEventHandler: (RecyclerViewUiEvents) -> Unit,
+
+) {
+    var value by remember {
+        mutableStateOf(fieldUiModel.value)
+    }
+
+    InputLink(
+        modifier = modifier.fillMaxWidth(),
+        title = fieldUiModel.label,
+        state = fieldUiModel.inputState(),
+        supportingText = fieldUiModel.supportingText(),
+        legendData = fieldUiModel.legend(),
+        inputText = value ?: "",
+        isRequiredField = fieldUiModel.mandatory,
+        onValueChanged = {
+            value = it
+            intentHandler(
+                FormIntent.OnSave(
```

Comment:
shouldn't be this FormIntent.OnTextChange?

Reply:
you are right, I'll correct it